core/src/components/datetime/utils/manipulation.ts

```diff
@@ -10,7 +10,14 @@ const fourDigit = (val: number | undefined): string => {
   return ('000' + (val !== undefined ? Math.abs(val) : '0')).slice(-4);
 };
 
-export const convertDataToISO = (data: any): string => {
+export function convertDataToISO(data: DatetimeParts): string;
+export function convertDataToISO(data: DatetimeParts[]): string[];
+export function convertDataToISO(data: DatetimeParts | DatetimeParts[]): string | string[];
```

Comment:
Is this line needed since the main function implementation has the same param/return types?

Reply:
Yep, same idea as with `parseDate`.